adafruit_funhouse/network.py

```diff
@@ -73,13 +73,12 @@ def __init__(
 
     def init_io_mqtt(self) -> IO_MQTT:
         """Initialize MQTT for Adafruit IO"""
-        try:
-            aio_username = self._get_setting("AIO_USERNAME")
-            aio_key = self._get_setting("AIO_KEY")
-        except KeyError:
-            raise KeyError(
-                "Adafruit IO secrets are kept in settings.toml, please add them there!\n\n"
-            ) from KeyError
+        aio_username = self._get_setting["ADAFRUIT_AIO_USERNAME"]
+        aio_key = self._get_setting["ADAFRUIT_AIO_KEY"]
```

Comment:
Will the addition of the `ADAFRUIT_` prefix to these keys require changing learn guide code or pages? Is that already coordinated if so?

Reply:
So PortalBase will look for either, and I'm working with Tyeth on the learn guides

Reply:
https://github.com/adafruit/Adafruit_CircuitPython_PortalBase/blob/53c1666ee15d1d811226a1fee79e4fd890936f42/adafruit_portalbase/network.py#L70 and https://github.com/adafruit/Adafruit_CircuitPython_PortalBase/blob/53c1666ee15d1d811226a1fee79e4fd890936f42/adafruit_portalbase/network.py#L137